nats-server/configuration/securing_nats/authorization.md

```diff
@@ -95,3 +113,18 @@ authorization: {
 }
 ```
 
+Here's an example with `allow_responses`:
+
+```text
+authorization: {
+	users: [
+		{ user: a, password: a },
+		{ user: b, password: b, permissions: {subscribe: "q", allow_responses: true } },
+		{ user: c, password: c, permissions: {subscribe: "q", allow_responses: { max: 5, expires: "1m" } } }
+	]
+}
+```
+
+User `a` has no restrictions. User `b` can listen on `q` for requests. If the received request has a reply subject it can reply at most one message. User `c` can also listen on `q` for requests, but is able to return at most 5 reply messages. The reply subject can be publish to for at most `1` minute.
```

Comment:
User 'b' can subscribe/listen on subject 'q', and by default can only publish messages to reply subjects. All other subjects will be denied. ...